cosmic-core/db-scripts/src/main/resources/db/schema-536to537-cleanup.sql

```diff
@@ -2,3 +2,6 @@
 -- Schema cleanup from 5.3.6 to 5.3.7;
 --
 
+DROP TABLE IF EXISTS `cloud`.`snapshot_policy`;
```

Comment:
It won't do much until the upgrade path is there. Since this file was already there (since KVM VM snapshotting feature) I also added it. This PR works fine without the tables being removed, and by committing it'll be automatically included in the next upgrade path. If you think it should go out, feel free to delete the commit.